core/divecomputer.cpp

```diff
@@ -287,11 +287,8 @@ void fixup_dc_duration(struct divecomputer &dc)
 
 static bool operator<(const event &ev1, const event &ev2)
 {
-	if (ev1.time.seconds < ev2.time.seconds)
-		return -1;
-	if (ev1.time.seconds > ev2.time.seconds)
-		return 1;
-	return ev1.name < ev2.name;
+	return std::tie(ev1.time.seconds, ev1.name) <
+	       std::tie(ev2.time.seconds, ev2.name);
```

Comment:
Spaces instead of tabs.

Reply:
I though it was standard practice to use tabs until indentation level and then fill with spaces, because that works with different tab settings?

Reply:
Yes, you are right.
I am used to using an extra tab for continuation lines in pretty much all other projects that I am working on, without worrying about alignment - this eliminates the noise from having to reshuffle spaces on lines with no other changes whenever the one line controlling the alignment changes.

Reply:
The code base is so inconsistent with respect to indentation (and coding style in general), that I think it doesn't matter one way or the other...